### PR TITLE
Unnecessary double quotes in  /libraries/joomla

### PR DESCRIPTION
**Pull request by frankmayer**
### Summary of Changes
- Replace unnecessary double quotes in /libraries/joomla
- Formatting

A lot of changes in this one, but they are only from double quotes to single quotes, where appropriate. 
Besides the quantity of the changes, this PR should be fairly easy to review. No change in behavior is expected.

### Testing Instructions

None, should not change behavior

### Documentation Changes Required

None.

### CODE REVIEW

File: libraries/joomla/database/driver/sqlsrv.php [hunk 1]
```diff
@@ -1004,7 +1004,7 @@ protected function checkFieldExists($table, $field)
 
 		$table = $this->replacePrefix((string) $table);
 		$query = "SELECT COLUMN_NAME FROM INFORMATION_SCHEMA.COLUMNS WHERE TABLE_NAME = '$table' AND COLUMN_NAME = '$field'" .
-			" ORDER BY ORDINAL_POSITION";
+			' ORDER BY ORDINAL_POSITION';
```
Comment (andrepereiradasilva) [inline libraries/joomla/database/driver/sqlsrv.php]: MHO is less readable if you have `'`and `"` delimiters in the same operation...
the same comment for all changes of this type in the PR

File: libraries/joomla/document/renderer/feed/rss.php [hunk 2]
```diff
@@ -70,69 +70,69 @@ public function render($name = '', $params = null, $content = null)
...
-		$feed .= "		<title>" . $feed_title . "</title>\n";
-		$feed .= "		<description><![CDATA[" . $data->getDescription() . "]]></description>\n";
-		$feed .= "		<link>" . str_replace(' ', '%20', $url . $datalink) . "</link>\n";
-		$feed .= "		<lastBuildDate>" . htmlspecialchars($now->toRFC822(true), ENT_COMPAT, 'UTF-8') . "</lastBuildDate>\n";
-		$feed .= "		<generator>" . $data->getGenerator() . "</generator>\n";
+		$feed .= '		<title>' . $feed_title . "</title>\n";
+		$feed .= '		<description><![CDATA[' . $data->getDescription() . "]]></description>\n";
+		$feed .= '		<link>' . str_replace(' ', '%20', $url . $datalink) . "</link>\n";
+		$feed .= '		<lastBuildDate>' . htmlspecialchars($now->toRFC822(true), ENT_COMPAT, 'UTF-8') . "</lastBuildDate>\n";
+		$feed .= '		<generator>' . $data->getGenerator() . "</generator>\n";
```
Comment (andrepereiradasilva) [inline libraries/joomla/document/renderer/feed/rss.php]: IMO this should be `"\t\t" . 'xxxxxxxx'` but that's other PR

File: libraries/joomla/session/session.php [hunk 3]
```diff
@@ -661,7 +661,13 @@ protected function _start()
 		// Temporary, PARTIAL, data migration of existing session data to avoid logout on update from J < 3.4.7
 		if (isset($_SESSION['__default']) && !empty($_SESSION['__default']))
 		{
-			$migratableKeys = array("user", "session.token", "session.counter", "session.timer.start", "session.timer.last", "session.timer.now");
+			$migratableKeys = array('user',
+									'session.token',
+									'session.counter',
+									'session.timer.start',
+									'session.timer.last',
+									'session.timer.now'
```
Comment (andrepereiradasilva) [inline libraries/joomla/session/session.php]: cs: please us the following as example for array with lot of values (notice just one tab and all values in the lines below)

```php
$myArray = array(
	'val1',
	'val2',
);
```

File: libraries/joomla/database/driver/sqlsrv.php (hunk 1 above)
Comment (frankmayer) [inline libraries/joomla/database/driver/sqlsrv.php]: Of course, my bad.

File: libraries/joomla/document/renderer/feed/rss.php (hunk 2 above)
Comment (frankmayer) [inline libraries/joomla/document/renderer/feed/rss.php]: Yes, that would be better.

File: libraries/joomla/database/query/postgresql.php [hunk 4]
```diff
@@ -612,11 +612,11 @@ public function dateAdd($date, $interval, $datePart)
 	{
 		if (substr($interval, 0, 1) != '-')
 		{
-			return "timestamp '" . $date . "' + interval '" . $interval . " " . $datePart . "'";
+			return "timestamp '" . $date . "' + interval '" . $interval . ' ' . $datePart . "'";
```
Comment (andrepereiradasilva) [inline libraries/joomla/database/query/postgresql.php]: same thing as the other. missing `'` and `"` IMHO makes the code less readable

File: libraries/joomla/database/query/postgresql.php [hunk 5]
```diff
@@ -612,11 +612,11 @@ public function dateAdd($date, $interval, $datePart)
...
 		}
 		else
 		{
-			return "timestamp '" . $date . "' - interval '" . ltrim($interval, '-') . " " . $datePart . "'";
+			return "timestamp '" . $date . "' - interval '" . ltrim($interval, '-') . ' ' . $datePart . "'";
```
Comment (andrepereiradasilva) [inline libraries/joomla/database/query/postgresql.php]: same thing as the other. missing `'` and `"` IMHO makes the code less readable

File: libraries/joomla/database/query/sqlite.php [hunk 6]
```diff
@@ -251,11 +251,11 @@ public function dateAdd($date, $interval, $datePart)
 
 		if (substr($interval, 0, 1) != '-')
 		{
-			return "datetime('" . $date . "', '+" . $interval . " " . $datePart . "')";
+			return "datetime('" . $date . "', '+" . $interval . ' ' . $datePart . "')";
```
Comment (andrepereiradasilva) [inline libraries/joomla/database/query/sqlite.php]: same thing as the other. missing `'` and `"` IMHO makes the code less readable

File: libraries/joomla/database/query/sqlite.php [hunk 7]
```diff
@@ -251,11 +251,11 @@ public function dateAdd($date, $interval, $datePart)
...
 		}
 		else
 		{
-			return "datetime('" . $date . "', '" . $interval . " " . $datePart . "')";
+			return "datetime('" . $date . "', '" . $interval . ' ' . $datePart . "')";
```
Comment (andrepereiradasilva) [inline libraries/joomla/database/query/sqlite.php]: same thing as the other. missing `'` and `"` IMHO makes the code less readable

File: libraries/joomla/document/renderer/feed/atom.php [hunk 8]
```diff
@@ -97,17 +97,17 @@ public function render($name = '', $params = null, $content = null)
 		}
 
 		$feed .= "	<link rel=\"alternate\" type=\"text/html\" href=\"" . $url . "\"/>\n";
-		$feed .= "	<id>" . str_replace(' ', '%20', $data->getBase()) . "</id>\n";
-		$feed .= "	<updated>" . htmlspecialchars($now->toISO8601(true), ENT_COMPAT, 'UTF-8') . "</updated>\n";
+		$feed .= '	<id>' . str_replace(' ', '%20', $data->getBase()) . "</id>\n";
+		$feed .= '	<updated>' . htmlspecialchars($now->toISO8601(true), ENT_COMPAT, 'UTF-8') . "</updated>\n";
```
Comment (andrepereiradasilva) [inline libraries/joomla/document/renderer/feed/atom.php]: same thing as the other. missing `'` and `"` IMHO makes the code less readable

File: libraries/joomla/document/renderer/feed/atom.php [hunk 9]
```diff
@@ -97,17 +97,17 @@ public function render($name = '', $params = null, $content = null)
...
 
 		if ($data->editor != '')
 		{
 			$feed .= "	<author>\n";
-			$feed .= "		<name>" . $data->editor . "</name>\n";
+			$feed .= '		<name>' . $data->editor . "</name>\n";
```
Comment (andrepereiradasilva) [inline libraries/joomla/document/renderer/feed/atom.php]: same thing as the other. missing `'` and `"` IMHO makes the code less readable

File: libraries/joomla/document/renderer/feed/atom.php [hunk 10]
```diff
@@ -97,17 +97,17 @@ public function render($name = '', $params = null, $content = null)
...
 
-			if ($data->editorEmail != "")
+			if ($data->editorEmail != '')
 			{
-				$feed .= "		<email>" . htmlspecialchars($data->editorEmail, ENT_COMPAT, 'UTF-8') . "</email>\n";
+				$feed .= '		<email>' . htmlspecialchars($data->editorEmail, ENT_COMPAT, 'UTF-8') . "</email>\n";
```
Comment (andrepereiradasilva) [inline libraries/joomla/document/renderer/feed/atom.php]: same thing as the other. missing `'` and `"` IMHO makes the code less readable

File: libraries/joomla/document/renderer/feed/atom.php [hunk 11]
```diff
@@ -120,7 +120,7 @@ public function render($name = '', $params = null, $content = null)
 			$versionHtmlEscaped = ' version="' . htmlspecialchars(JVersion::RELEASE, ENT_COMPAT, 'UTF-8') . '"';
 		}
 
-		$feed .= "	<generator uri=\"https://www.joomla.org\"" . $versionHtmlEscaped . ">" . $data->getGenerator() . "</generator>\n";
+		$feed .= "	<generator uri=\"https://www.joomla.org\"" . $versionHtmlEscaped . '>' . $data->getGenerator() . "</generator>\n";
```
Comment (andrepereiradasilva) [inline libraries/joomla/document/renderer/feed/atom.php]: same thing as the other. missing `'` and `"` IMHO makes the code less readable

File: libraries/joomla/document/renderer/feed/atom.php [hunk 12]
```diff
@@ -129,11 +129,11 @@ public function render($name = '', $params = null, $content = null)
 
 			if (preg_match('/[\x80-\xFF]/', $itemlink))
 			{
-				$itemlink = implode("/", array_map("rawurlencode", explode("/", $itemlink)));
+				$itemlink = implode('/', array_map('rawurlencode', explode('/', $itemlink)));
 			}
 
 			$feed .= "	<entry>\n";
-			$feed .= "		<title>" . htmlspecialchars(strip_tags($data->items[$i]->title), ENT_COMPAT, 'UTF-8') . "</title>\n";
+			$feed .= '		<title>' . htmlspecialchars(strip_tags($data->items[$i]->title), ENT_COMPAT, 'UTF-8') . "</title>\n";
```
Comment (andrepereiradasilva) [inline libraries/joomla/document/renderer/feed/atom.php]: same thing as the other. missing `'` and `"` IMHO makes the code less readable

File: libraries/joomla/document/renderer/feed/atom.php [hunk 13]
```diff
@@ -143,8 +143,8 @@ public function render($name = '', $params = null, $content = null)
 
 			$itemDate = JFactory::getDate($data->items[$i]->date);
 			$itemDate->setTimeZone($tz);
-			$feed .= "		<published>" . htmlspecialchars($itemDate->toISO8601(true), ENT_COMPAT, 'UTF-8') . "</published>\n";
-			$feed .= "		<updated>" . htmlspecialchars($itemDate->toISO8601(true), ENT_COMPAT, 'UTF-8') . "</updated>\n";
+			$feed .= '		<published>' . htmlspecialchars($itemDate->toISO8601(true), ENT_COMPAT, 'UTF-8') . "</published>\n";
+			$feed .= '		<updated>' . htmlspecialchars($itemDate->toISO8601(true), ENT_COMPAT, 'UTF-8') . "</updated>\n";
```
Comment (andrepereiradasilva) [inline libraries/joomla/document/renderer/feed/atom.php]: same thing as the other. missing `'` and `"` IMHO makes the code less readable

File: libraries/joomla/document/renderer/feed/atom.php [hunk 14]
```diff
@@ -155,16 +155,16 @@ public function render($name = '', $params = null, $content = null)
 				$itemGuid = htmlspecialchars($data->items[$i]->guid, ENT_COMPAT, 'UTF-8');
 			}
 
-			$feed .= "		<id>" . $itemGuid . "</id>\n";
+			$feed .= '		<id>' . $itemGuid . "</id>\n";
```
Comment (andrepereiradasilva) [inline libraries/joomla/document/renderer/feed/atom.php]: same thing as the other. missing `'` and `"` IMHO makes the code less readable

File: libraries/joomla/document/renderer/feed/atom.php [hunk 15]
```diff
@@ -155,16 +155,16 @@ public function render($name = '', $params = null, $content = null)
...
 
-			if ($data->items[$i]->author != "")
+			if ($data->items[$i]->author != '')
 			{
 				$feed .= "		<author>\n";
-				$feed .= "			<name>" . htmlspecialchars($data->items[$i]->author, ENT_COMPAT, 'UTF-8') . "</name>\n";
+				$feed .= '			<name>' . htmlspecialchars($data->items[$i]->author, ENT_COMPAT, 'UTF-8') . "</name>\n";
```
Comment (andrepereiradasilva) [inline libraries/joomla/document/renderer/feed/atom.php]: same thing as the other. missing `'` and `"` IMHO makes the code less readable

File: libraries/joomla/document/renderer/feed/atom.php [hunk 16]
```diff
@@ -155,16 +155,16 @@ public function render($name = '', $params = null, $content = null)
...
 
 				if (!empty($data->items[$i]->authorEmail))
 				{
-					$feed .= "			<email>" . htmlspecialchars($data->items[$i]->authorEmail, ENT_COMPAT, 'UTF-8') . "</email>\n";
+					$feed .= '			<email>' . htmlspecialchars($data->items[$i]->authorEmail, ENT_COMPAT, 'UTF-8') . "</email>\n";
```
Comment (andrepereiradasilva) [inline libraries/joomla/document/renderer/feed/atom.php]: same thing as the other. missing `'` and `"` IMHO makes the code less readable

File: libraries/joomla/document/renderer/feed/rss.php (hunk 2 above)
Comment (andrepereiradasilva) [inline libraries/joomla/document/renderer/feed/rss.php]: same thing as the other. missing `'` and `"` IMHO makes the code less readable

File: libraries/joomla/document/renderer/feed/rss.php [hunk 17]
```diff
@@ -70,69 +70,69 @@ public function render($name = '', $params = null, $content = null)
...
 
 		if ($data->image != null)
 		{
 			$feed .= "		<image>\n";
-			$feed .= "			<url>" . $data->image->url . "</url>\n";
-			$feed .= "			<title>" . htmlspecialchars($data->image->title, ENT_COMPAT, 'UTF-8') . "</title>\n";
-			$feed .= "			<link>" . str_replace(' ', '%20', $data->image->link) . "</link>\n";
+			$feed .= '			<url>' . $data->image->url . "</url>\n";
+			$feed .= '			<title>' . htmlspecialchars($data->image->title, ENT_COMPAT, 'UTF-8') . "</title>\n";
+			$feed .= '			<link>' . str_replace(' ', '%20', $data->image->link) . "</link>\n";
```
Comment (andrepereiradasilva) [inline libraries/joomla/document/renderer/feed/rss.php]: same thing as the other. missing `'` and `"` IMHO makes the code less readable

File: libraries/joomla/document/renderer/feed/rss.php [hunk 18]
```diff
@@ -70,69 +70,69 @@ public function render($name = '', $params = null, $content = null)
...
 
 			if ($data->image->width != '')
 			{
-				$feed .= "			<width>" . $data->image->width . "</width>\n";
+				$feed .= '			<width>' . $data->image->width . "</width>\n";
```
Comment (andrepereiradasilva) [inline libraries/joomla/document/renderer/feed/rss.php]: same thing as the other. missing `'` and `"` IMHO makes the code less readable

File: libraries/joomla/document/renderer/feed/rss.php [hunk 19]
```diff
@@ -70,69 +70,69 @@ public function render($name = '', $params = null, $content = null)
...
 			}
 
 			if ($data->image->height != '')
 			{
-				$feed .= "			<height>" . $data->image->height . "</height>\n";
+				$feed .= '			<height>' . $data->image->height . "</height>\n";
```
Comment (andrepereiradasilva) [inline libraries/joomla/document/renderer/feed/rss.php]: same thing as the other. missing `'` and `"` IMHO makes the code less readable

File: libraries/joomla/document/renderer/feed/rss.php [hunk 20]
```diff
@@ -70,69 +70,69 @@ public function render($name = '', $params = null, $content = null)
...
 			}
 
 			if ($data->image->description != '')
 			{
-				$feed .= "			<description><![CDATA[" . $data->image->description . "]]></description>\n";
+				$feed .= '			<description><![CDATA[' . $data->image->description . "]]></description>\n";
```
Comment (andrepereiradasilva) [inline libraries/joomla/document/renderer/feed/rss.php]: same thing as the other. missing `'` and `"` IMHO makes the code less readable

File: libraries/joomla/document/renderer/feed/rss.php [hunk 21]
```diff
@@ -70,69 +70,69 @@ public function render($name = '', $params = null, $content = null)
...
 			}
 
 			$feed .= "		</image>\n";
 		}
 
 		if ($data->getLanguage() !== '')
 		{
-			$feed .= "		<language>" . $data->getLanguage() . "</language>\n";
+			$feed .= '		<language>' . $data->getLanguage() . "</language>\n";
```
Comment (andrepereiradasilva) [inline libraries/joomla/document/renderer/feed/rss.php]: same thing as the other. missing `'` and `"` IMHO makes the code less readable

File: libraries/joomla/document/renderer/feed/rss.php [hunk 22]
```diff
@@ -70,69 +70,69 @@ public function render($name = '', $params = null, $content = null)
...
 		}
 
 		if ($data->copyright != '')
 		{
-			$feed .= "		<copyright>" . htmlspecialchars($data->copyright, ENT_COMPAT, 'UTF-8') . "</copyright>\n";
+			$feed .= '		<copyright>' . htmlspecialchars($data->copyright, ENT_COMPAT, 'UTF-8') . "</copyright>\n";
```
Comment (andrepereiradasilva) [inline libraries/joomla/document/renderer/feed/rss.php]: same thing as the other. missing `'` and `"` IMHO makes the code less readable

File: libraries/joomla/document/renderer/feed/rss.php [hunk 23]
```diff
@@ -70,69 +70,69 @@ public function render($name = '', $params = null, $content = null)
...
 		}
 
 		if ($data->editorEmail != '')
 		{
-			$feed .= "		<managingEditor>" . htmlspecialchars($data->editorEmail, ENT_COMPAT, 'UTF-8') . ' ('
+			$feed .= '		<managingEditor>' . htmlspecialchars($data->editorEmail, ENT_COMPAT, 'UTF-8') . ' ('
```
Comment (andrepereiradasilva) [inline libraries/joomla/document/renderer/feed/rss.php]: same thing as the other. missing `'` and `"` IMHO makes the code less readable

File: libraries/joomla/document/renderer/feed/rss.php [hunk 24]
```diff
@@ -70,69 +70,69 @@ public function render($name = '', $params = null, $content = null)
...
 				. htmlspecialchars($data->editor, ENT_COMPAT, 'UTF-8') . ")</managingEditor>\n";
 		}
 
 		if ($data->webmaster != '')
 		{
-			$feed .= "		<webMaster>" . htmlspecialchars($data->webmaster, ENT_COMPAT, 'UTF-8') . "</webMaster>\n";
+			$feed .= '		<webMaster>' . htmlspecialchars($data->webmaster, ENT_COMPAT, 'UTF-8') . "</webMaster>\n";
```
Comment (andrepereiradasilva) [inline libraries/joomla/document/renderer/feed/rss.php]: same thing as the other. missing `'` and `"` IMHO makes the code less readable

File: libraries/joomla/document/renderer/feed/rss.php [hunk 25]
```diff
@@ -70,69 +70,69 @@ public function render($name = '', $params = null, $content = null)
...
 		}
 
 		if ($data->pubDate != '')
 		{
 			$pubDate = JFactory::getDate($data->pubDate);
 			$pubDate->setTimeZone($tz);
-			$feed .= "		<pubDate>" . htmlspecialchars($pubDate->toRFC822(true), ENT_COMPAT, 'UTF-8') . "</pubDate>\n";
+			$feed .= '		<pubDate>' . htmlspecialchars($pubDate->toRFC822(true), ENT_COMPAT, 'UTF-8') . "</pubDate>\n";
```
Comment (andrepereiradasilva) [inline libraries/joomla/document/renderer/feed/rss.php]: same thing as the other. missing `'` and `"` IMHO makes the code less readable

File: libraries/joomla/document/renderer/feed/rss.php [hunk 26]
```diff
@@ -141,38 +141,38 @@ public function render($name = '', $params = null, $content = null)
 			{
 				foreach ($data->category as $cat)
 				{
-					$feed .= "		<category>" . htmlspecialchars($cat, ENT_COMPAT, 'UTF-8') . "</category>\n";
+					$feed .= '		<category>' . htmlspecialchars($cat, ENT_COMPAT, 'UTF-8') . "</category>\n";
```
Comment (andrepereiradasilva) [inline libraries/joomla/document/renderer/feed/rss.php]: same thing as the other. missing `'` and `"` IMHO makes the code less readable

File: libraries/joomla/document/renderer/feed/rss.php [hunk 27]
```diff
@@ -141,38 +141,38 @@ public function render($name = '', $params = null, $content = null)
...
 				}
 			}
 			else
 			{
-				$feed .= "		<category>" . htmlspecialchars($data->category, ENT_COMPAT, 'UTF-8') . "</category>\n";
+				$feed .= '		<category>' . htmlspecialchars($data->category, ENT_COMPAT, 'UTF-8') . "</category>\n";
```
Comment (andrepereiradasilva) [inline libraries/joomla/document/renderer/feed/rss.php]: same thing as the other. missing `'` and `"` IMHO makes the code less readable

File: libraries/joomla/document/renderer/feed/rss.php [hunk 28]
```diff
@@ -141,38 +141,38 @@ public function render($name = '', $params = null, $content = null)
...
 			}
 		}
 
 		if ($data->docs != '')
 		{
-			$feed .= "		<docs>" . htmlspecialchars($data->docs, ENT_COMPAT, 'UTF-8') . "</docs>\n";
+			$feed .= '		<docs>' . htmlspecialchars($data->docs, ENT_COMPAT, 'UTF-8') . "</docs>\n";
```
Comment (andrepereiradasilva) [inline libraries/joomla/document/renderer/feed/rss.php]: same thing as the other. missing `'` and `"` IMHO makes the code less readable

File: libraries/joomla/document/renderer/feed/rss.php [hunk 29]
```diff
@@ -141,38 +141,38 @@ public function render($name = '', $params = null, $content = null)
...
 		}
 
 		if ($data->ttl != '')
 		{
-			$feed .= "		<ttl>" . htmlspecialchars($data->ttl, ENT_COMPAT, 'UTF-8') . "</ttl>\n";
+			$feed .= '		<ttl>' . htmlspecialchars($data->ttl, ENT_COMPAT, 'UTF-8') . "</ttl>\n";
```
Comment (andrepereiradasilva) [inline libraries/joomla/document/renderer/feed/rss.php]: same thing as the other. missing `'` and `"` IMHO makes the code less readable

File: libraries/joomla/document/renderer/feed/rss.php [hunk 30]
```diff
@@ -141,38 +141,38 @@ public function render($name = '', $params = null, $content = null)
...
 		}
 
 		if ($data->rating != '')
 		{
-			$feed .= "		<rating>" . htmlspecialchars($data->rating, ENT_COMPAT, 'UTF-8') . "</rating>\n";
+			$feed .= '		<rating>' . htmlspecialchars($data->rating, ENT_COMPAT, 'UTF-8') . "</rating>\n";
```
Comment (andrepereiradasilva) [inline libraries/joomla/document/renderer/feed/rss.php]: same thing as the other. missing `'` and `"` IMHO makes the code less readable

File: libraries/joomla/document/renderer/feed/rss.php [hunk 31]
```diff
@@ -141,38 +141,38 @@ public function render($name = '', $params = null, $content = null)
...
 		}
 
 		if ($data->skipHours != '')
 		{
-			$feed .= "		<skipHours>" . htmlspecialchars($data->skipHours, ENT_COMPAT, 'UTF-8') . "</skipHours>\n";
+			$feed .= '		<skipHours>' . htmlspecialchars($data->skipHours, ENT_COMPAT, 'UTF-8') . "</skipHours>\n";
```
Comment (andrepereiradasilva) [inline libraries/joomla/document/renderer/feed/rss.php]: same thing as the other. missing `'` and `"` IMHO makes the code less readable

File: libraries/joomla/document/renderer/feed/rss.php [hunk 32]
```diff
@@ -141,38 +141,38 @@ public function render($name = '', $params = null, $content = null)
...
 		}
 
 		if ($data->skipDays != '')
 		{
-			$feed .= "		<skipDays>" . htmlspecialchars($data->skipDays, ENT_COMPAT, 'UTF-8') . "</skipDays>\n";
+			$feed .= '		<skipDays>' . htmlspecialchars($data->skipDays, ENT_COMPAT, 'UTF-8') . "</skipDays>\n";
```
Comment (andrepereiradasilva) [inline libraries/joomla/document/renderer/feed/rss.php]: same thing as the other. missing `'` and `"` IMHO makes the code less readable

File: libraries/joomla/document/renderer/feed/rss.php [hunk 33]
```diff
@@ -190,8 +190,8 @@ public function render($name = '', $params = null, $content = null)
 			}
 
 			$feed .= "		<item>\n";
-			$feed .= "			<title>" . htmlspecialchars(strip_tags($data->items[$i]->title), ENT_COMPAT, 'UTF-8') . "</title>\n";
-			$feed .= "			<link>" . str_replace(' ', '%20', $itemlink) . "</link>\n";
+			$feed .= '			<title>' . htmlspecialchars(strip_tags($data->items[$i]->title), ENT_COMPAT, 'UTF-8') . "</title>\n";
+			$feed .= '			<link>' . str_replace(' ', '%20', $itemlink) . "</link>\n";
```
Comment (andrepereiradasilva) [inline libraries/joomla/document/renderer/feed/rss.php]: same thing as the other. missing `'` and `"` IMHO makes the code less readable

File: libraries/joomla/document/renderer/feed/rss.php [hunk 34]
```diff
@@ -202,11 +202,11 @@ public function render($name = '', $params = null, $content = null)
 				$feed .= "			<guid isPermaLink=\"false\">" . htmlspecialchars($data->items[$i]->guid, ENT_COMPAT, 'UTF-8') . "</guid>\n";
 			}
 
-			$feed .= "			<description><![CDATA[" . $this->_relToAbs($data->items[$i]->description) . "]]></description>\n";
+			$feed .= '			<description><![CDATA[' . $this->_relToAbs($data->items[$i]->description) . "]]></description>\n";
```
Comment (andrepereiradasilva) [inline libraries/joomla/document/renderer/feed/rss.php]: same thing as the other. missing `'` and `"` IMHO makes the code less readable

File: libraries/joomla/document/renderer/feed/rss.php [hunk 35]
```diff
@@ -224,25 +224,25 @@ public function render($name = '', $params = null, $content = null)
 				{
 					foreach ($data->items[$i]->category as $cat)
 					{
-						$feed .= "			<category>" . htmlspecialchars($cat, ENT_COMPAT, 'UTF-8') . "</category>\n";
+						$feed .= '			<category>' . htmlspecialchars($cat, ENT_COMPAT, 'UTF-8') . "</category>\n";
```
Comment (andrepereiradasilva) [inline libraries/joomla/document/renderer/feed/rss.php]: same thing as the other. missing `'` and `"` IMHO makes the code less readable

File: libraries/joomla/document/renderer/feed/rss.php [hunk 36]
```diff
@@ -224,25 +224,25 @@ public function render($name = '', $params = null, $content = null)
...
 					}
 				}
 				else
 				{
-					$feed .= "			<category>" . htmlspecialchars($data->items[$i]->category, ENT_COMPAT, 'UTF-8') . "</category>\n";
+					$feed .= '			<category>' . htmlspecialchars($data->items[$i]->category, ENT_COMPAT, 'UTF-8') . "</category>\n";
```
Comment (andrepereiradasilva) [inline libraries/joomla/document/renderer/feed/rss.php]: same thing as the other. missing `'` and `"` IMHO makes the code less readable

File: libraries/joomla/document/renderer/feed/rss.php [hunk 37]
```diff
@@ -224,25 +224,25 @@ public function render($name = '', $params = null, $content = null)
...
 				}
 			}
 
 			if ($data->items[$i]->comments != '')
 			{
-				$feed .= "			<comments>" . htmlspecialchars($data->items[$i]->comments, ENT_COMPAT, 'UTF-8') . "</comments>\n";
+				$feed .= '			<comments>' . htmlspecialchars($data->items[$i]->comments, ENT_COMPAT, 'UTF-8') . "</comments>\n";
```
Comment (andrepereiradasilva) [inline libraries/joomla/document/renderer/feed/rss.php]: same thing as the other. missing `'` and `"` IMHO makes the code less readable

File: libraries/joomla/document/renderer/feed/rss.php [hunk 38]
```diff
@@ -224,25 +224,25 @@ public function render($name = '', $params = null, $content = null)
...
 			}
 
 			if ($data->items[$i]->date != '')
 			{
 				$itemDate = JFactory::getDate($data->items[$i]->date);
 				$itemDate->setTimeZone($tz);
-				$feed .= "			<pubDate>" . htmlspecialchars($itemDate->toRFC822(true), ENT_COMPAT, 'UTF-8') . "</pubDate>\n";
+				$feed .= '			<pubDate>' . htmlspecialchars($itemDate->toRFC822(true), ENT_COMPAT, 'UTF-8') . "</pubDate>\n";
```
Comment (andrepereiradasilva) [inline libraries/joomla/document/renderer/feed/rss.php]: same thing as the other. missing `'` and `"` IMHO makes the code less readable

File: libraries/joomla/database/query/postgresql.php (hunk 4 above)
Comment (frankmayer) [inline libraries/joomla/database/query/postgresql.php]: How about doing this code then the following way? (makes it even easier to read...)
```
	public function dateAdd($date, $interval, $datePart)
	{
		if (substr($interval, 0, 1) != '-')
		{
			return "timestamp '$date' + interval '$interval $datePart'";
		}
		else
		{   
                        $interval = ltrim($interval, '-');
			return "timestamp '$date' - interval '$interval $datePart'";
		}
	}

```

**Review by andrepereiradasilva — APPROVED** [review]

**Comment by anibalsanchez** [comment]
I have tested this item :white_check_mark: successfully on 6617b61967077cb5af725a13daf28b27e45b6f2c<br /><br />Code review OK<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/13372">issues.joomla.org/tracker/joomla-cms/13372</a>.</sub>

**Pull request merged**